CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - "You miss 100 percent of the chances you don't take. — Wayne Gretzky" — Michael Scott
 - [react] feat: Add @sentry/react package (#2631)
 - [browser] Change XHR instrumentation order to handle `onreadystatechange` breadcrumbs correctly (#2643)
+- [browser] feat: Report LCP metric on pageload transactions (#2624)
```

Comment:
```suggestion
- [apm] feat: Report LCP metric on pageload transactions (#2624)
```

Reply:
What's the logic to choose this marker in front of the change log lines? This one change was not obvious to me, it is related to `apm`, but it only applies to users of `@sentry/browser` and not of `@sentry/node`.